Resources/Private/Scss/_featurebox.scss

```diff
@@ -0,0 +1,111 @@
+.featureSlider {
+    @include media-l {
+        width: width(l-12);
+    }
+
+    @include media-m {
+        width: width(m-12);
+    }
+
+    @include media-s {
+        width: width(s-6);
+    }
+    justify-content: flex-start;
+    padding-bottom: $grid-l-gutter;
```

Comment:
a variable related to grid shouldn't be used with paddings (or anything besides the grid implemantation)